tools/rna_tools/targetfinder/.shed.yml

```diff
@@ -0,0 +1,9 @@
+name: targetfinder
+owner: dyusuf
```

Comment:
owner: rna_team
This is a shared account between a few peoples that are interested in RNA and Galaxy.